packages/cli/lib/discoverManifests.js

```diff
@@ -35,14 +36,18 @@ export default async function * () {
   }
 }
 
-async function getInstalledPackages() {
+async function getInstalledPackages(all) {
   if (isInstalledGlobally) {
+    let npmList = 'npm list -g'
+    if (all) {
+      npmList += ' --all'
+    }
```

Comment:
When resolving steps for shorthand syntax, we need all packages, not inly those installed directly